index.html

```diff
@@ -972,12 +973,14 @@
       <li>
         Writing <a>Web NFC content</a> to an <a>NFC tag</a> does not need to
         <a>obtain permission</a>, if the <a>NFC tag</a> contains
-        <a>trusted integrity NFC content</a> and the value of the <a>NDEF Id</a>
-        field is equal to the <a>ASCII serialized origin</a> of the
-        <a>incumbent settings object</a>.
+        <a>trusted integrity NFC content</a> which is a <a>Web NFC message</a>
```

Comment:
This would read better as "if the NFC tag contains trusted integrity NFC content, and that content is a Web NFC message whose origin is equal to the ASCII serialized origin of the incumbent settings object."

That'll require you to define the "origin" of a Web NFC message in https://rawgit.com/zolkis/web-nfc/gh-pages/index.html#dfn-web-nfc-message, but doing so is straightforward.